app/routes/index.js

```diff
@@ -1,4 +1,7 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  redirect: function(){
+    this.transitionTo("/air_quality_observations/27601");
```

Comment:
I actually do not know how to do that in ember and I don't have to time to lookup how :disappointed: 

Gonna merge so I can deploy before EOD but adding a FIXME so that future me changes this